src/services/users.js

```diff
@@ -182,5 +185,118 @@ class UsersService {
       res.boom.badImplementation();
     }
   }
+
+  async addProducts (req, res) {
+    try {
+      const user = await User.findById(req.session.passport.user.id).exec();
```

Comment:
Create method getUserById inside users service and call it from here. Lets follow pattern: each service responsible for it's model

Reply:
but that is in users service, which is responsible for users model, wouldn't creating this method be unnecessary?

Reply:
Sorry, i confused the PR's) Everything ok)

Reply:
But anyway, this part of code is in every method, so move it in separate one

Reply:
I actually realized that we don't need to check if a user exists because we are logged as this user. if the user didn't exist there was no way of logging in.